nf_core/lint_utils.py

```diff
@@ -73,6 +73,10 @@ def run_prettier_on_file(file):
     except subprocess.CalledProcessError as e:
         if ": SyntaxError: " in e.stdout.decode():
             log.critical(f"Can't format {file} because it has a syntax error.\n{e.stdout.decode()}")
+        if "files were modified by this hook" in e.stdout.decode():
+            all_lines = [line for line in e.stdout.decode().split("\n")]
+            files = "\n".join(all_lines[3:])
+            log.info(f"The following files were modified by prettier:\n {files}")
```

Comment:
I don't think that the end user needs to know or care that this has happened, most of the time.

```suggestion
            log.debug(f"The following files were modified by prettier:\n {files}")
```

Reply:
And if we're going to go down the debug logging road, could also add an `else` after the `except` to debug log that prettier didn't want to change anything..

Reply:
Logging on debug sounds good, I am getting this message with every single command, which seems too noisy